DGC.schema.json

```diff
@@ -29,7 +29,7 @@
       "description": "Date of Birth of the person addressed in the DGC. ISO 8601 date format restricted to range 1900-2099",
       "type": "string",
       "format": "date",
-      "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}",
+      "pattern": "^(19|20)\\d{2}",
```

Comment:
A date can never be year only with format `date`. If this change is required we must:

1. remove the format
2. fix the pattern so it allows either YYYY or YYYY-MM-DD
3. bump the schema version to 2.0.0

Reply:
> A date can never be year only with format `date`. 

Yes it can.

> If this change is required we must:
> 
> 1. remove the format

Possible, acceptable.


> 2. fix the pattern so it allows either YYYY or YYYY-MM-DD

Broken if you only have YYYY-MM, which is also valid.


> 3. bump the schema version to 2.0.0

Why 2.0.0? It does not break back-compatibility.